classes/AlertView/Standard.php

```diff
@@ -454,7 +459,20 @@ private function searchForConstituenciesAndMembers() {
                         $MEMBER = new \MySociety\TheyWorkForYou\Member(['constituency' => $constituency, 'house' => $house]);
                         $cons[] = [ 'member' => $MEMBER, 'constituency' => $constituency ];
                     }
-                } catch (\MySociety\TheyWorkForYou\MySociety\TheyWorkForYou\MemberException $e) {
+                } catch (\MySociety\TheyWorkForYou\MemberException $e) {
+                    // do nothing
+                }
+            }
+            $this->data['constituencies'] = $cons;
+        }
+
```

Comment:
Something not right here - so if there's match on constituency name (`constituencies`) we loop through and find all the matching people, but then if there was a `member_constituencies` match (which presumably there would also be if constituencies matched), we throw all that away and do it all again?